src/Microsoft.Azure.SignalR.AspNet/ClientConnections/ClientConnectionManager.cs

```diff
@@ -119,8 +127,18 @@ internal static string GetContentAndDispose(MemoryStream stream)
             {
                 return reader.ReadToEnd();
             }
-        }
-
+        }
+
+        public async Task AllConnectionsCompleted()
+        {
+            List<Task> tasks = new List<Task>();
+            foreach (var tcs in _clientCompletedTcs.Values)
+            {
+                tasks.Add(tcs.Task);
+            }
+            await Task.WhenAll(tasks);

```

Comment:
`return Task.WhenAll(_clientCompletedTcs.Select(s=>s.Value.Task))`

Reply:
I don't know why I can't do this in my `VS`?

I've been told that no definition for `Select`, for both `ConcurrentDictionary` and `ICollections` (value)

Reply:
Problem solved by adding "using System.Linq"